sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -85,3 +85,23 @@
 --    arguments: none
 --    behavior: can take a long time.
 
+
+
+#IfMissingColumn list_options edit_options
+  ALTER TABLE `list_options` ADD `edit_options` TINYINT(1) NOT NULL DEFAULT '1';
+#Endif
+
+#IfNotTable zf_multiple_db
+  CREATE TABLE `zf_multiple_db` (
+    `id` int(11) NOT NULL,
+    `namespace` varchar(255) NOT NULL,
+    `username` varchar(255) NOT NULL,
+    `password` text NOT NULL,
```

Comment:
make text bare or will break on modern mysql(note text can not be null and is always non null):
``````
`password` text,
``````